Source/Model/User/ZMUser+Permissions.swift

```diff
@@ -130,27 +130,27 @@ public extension ZMUser {
     @objc(canModifyAccessControlSettingsInConversation:)
     func canModifyAccessControlSettings(in conversation: ZMConversation) -> Bool {
         
-        if conversation.conversationType == .group {
+        // Check conversation
+        guard conversation.conversationType == .group,
+            let moc = self.managedObjectContext,
+            team?.remoteIdentifier != nil
+            else { return false }
+        
+        if conversation.conversationType == .group && team?.remoteIdentifier != nil {
```

Comment:
team remote id is checked above.

```suggestion
        if conversation.conversationType == .group {
```